samples/CosmosDbIntegtation/ReadThroughSamples/ListsSample.cs

```diff
@@ -0,0 +1,105 @@
+﻿using Microsoft.Azure.Cosmos;
+using Microsoft.Azure.Cosmos.Linq;
+using Microsoft.Azure.Functions.Worker.Extensions.Redis;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using StackExchange.Redis;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Policy;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Redis.Samples
+{
+    public static class ListsSample
```

Comment:
Do you think it would make sense to rename this (and the other classes) to `ListSample`? I guess we are only working with one list, so it might make more sense. Also it aligns with the other PRs which are `StreamSample` and ``PubSubSample`.

Reply:
Thanks!